app/controllers/admin/tenants_controller.rb

```diff
@@ -32,4 +33,20 @@ def update
     def tenant_params
       params.require(:tenant).permit(:name, :schema, :schema_type)
     end
+
+    def create_tenant_administrator
+      Tenant.switch(@tenant.schema) do
+        admin = User.create!(username: current_user.username,
+                             email: current_user.email,
+                             password: "password_tmp",
+                             password_confirmation: "password_tmp",
+                             confirmed_at: Time.current,
+                             terms_of_service: "1")
+        admin.update!(password: nil,
+                      password_confirmation: nil,
+                      encrypted_password: current_user.encrypted_password)
```

Comment:
I wonder whether it's clear that if the administrator changes the password in the main tenant, the password in the new tenant remains unchanged :thinking:.

Reply:
I updated the notice text so it better explains the feature behaviour.